### PR TITLE
add quotes for uppercase table names

### PR DESCRIPTION
**Pull request by katherinebernstein**

### CODE REVIEW

**Review by makosblade — DISMISSED** [review]

**Review by makosblade — APPROVED** [review]

File: indexer/src/main/kotlin/com/openlattice/BackgroundExternalDatabaseSyncingService.kt [hunk 1]
```diff
@@ -109,14 +109,21 @@ class BackgroundExternalDatabaseSyncingService(
...
         } finally {
             taskLock.unlock()
         }
     }
 
     private fun syncOrganizationDatabases(orgId: UUID): Int {
         var totalSynced = 0
-        val dbName = organizationDatabases.getValue(orgId).name
+        val maybeOrg = organizationDatabases[orgId]
+        if ( maybeOrg == null ){
```
Comment (UnsungHero97) [inline indexer/src/main/kotlin/com/openlattice/BackgroundExternalDatabaseSyncingService.kt:122]: Please fix formatting.

File: settings.gradle.kts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 pluginManagement {
     plugins {
-        kotlin("jvm")                                   version "1.3.72" apply false
+        kotlin("jvm")                               version "1.3.72" apply false
```
Comment (UnsungHero97) [inline settings.gradle.kts:3]: Please fix formatting.

Comment (UnsungHero97) [inline settings.gradle.kts:3]: Please fix formatting.

**Review by UnsungHero97 — APPROVED** [review]

File: settings.gradle.kts (hunk 2 above)
Comment (katherinebernstein) [reply settings.gradle.kts:3]: updated on https://github.com/openlattice/openlattice/pull/59

**Pull request merged**